pkg/proxy/service/cluster_service.go

```diff
@@ -43,7 +33,7 @@ func NewMemberClusterService(context servicecontext.ServiceContext, options ...O
 	return si
 }
 
-func (s *ServiceImpl) GetNamespace(ctx *gin.Context, userID, username string) (*namespace.NamespaceAccess, error) {
+func (s *ServiceImpl) GetClusterAccess(ctx *gin.Context, userID, username string) (*access.ClusterAccess, error) {
 	// Get Signup
 	signup, err := s.ServiceContext.Services().SignupService().GetSignup(userID, username)
```

Comment:
it's probably fine for now, but keep in mind that the `GetSignup` function sets the cluster-name based on the value that is in MUR, but we should ideally take it from the Space CR.

Reply:
I guess we'll need to change the cluster lookup logic anyway so I think we can change it then?